apps/passport-server/src/services/semaphore.ts

```diff
@@ -41,6 +44,7 @@ export class SemaphoreService {
     const ps = await fetchPassportParticipants(dbPool);
     console.log(`[SEMA] Rebuilding groups, ${ps.length} total participants.`);
     this.participants = {};
+    this.groupParticipants = new Group("1", 16);
     this.groupResidents = new Group("1", 16);
     this.groupVisitors = new Group("1", 16);
```

Comment:
should the ids here be updated?

Reply:
good call. refactored to remove duplication